docs/HowTo/Troubleshoot/Trace-Transactions.md

```diff
@@ -5,25 +5,44 @@ description: How to trace transactions
 # Trace transactions
 
 To get detailed information about transaction processing, use the
-[`TRACE` API](../../Reference/API-Methods.md#trace-methods). Enable the
-[`TRACE` API](../../Reference/API-Methods.md#trace-methods) using the
+[`TRACE` API](../../Reference/API-Methods.md#trace-methods).
+Enable the `TRACE` API using the
 [`--rpc-http-api`](../../Reference/CLI/CLI-Syntax.md#rpc-http-api) or
 [`--rpc-ws-api`](../../Reference/CLI/CLI-Syntax.md#rpc-ws-api) command line options.
 
-To specify the trace types required and retrieve the transaction traces for a block, use
-[`trace_replayBlockTransactions`](../../Reference/API-Methods.md#trace_replayblocktransactions). Options are
-[`trace`, `vmTrace`, or `stateDiff`](../../Reference/Trace-Types.md).
+The `TRACE` API has two sets of trace calls, [ad-hoc tracing APIs](#ad-hoc-tracing-apis) and
+[transaction-trace filtering APIs](#transaction-trace-filtering-apis).
 
-To retrieve the [`trace` type](../../Reference/Trace-Types.md#trace) for a specific:
+## Ad-hoc tracing APIs
 
-* Block, use [`trace_block`](../../Reference/API-Methods.md#trace_block).
-* Transaction, use [`trace_transaction`](../../Reference/API-Methods.md#trace_transaction).
+These APIs allow different diagnostic options when tracing calls or transactions.
+The options are [`trace`, `vmTrace`, or `stateDiff`](../../Reference/Trace-Types.md).
 
-Your node must be an archive node (that is, synchronized without pruning or fast sync) or the
-requested block or transaction must be within the [the number of pruning blocks retained](../../Reference/CLI/CLI-Syntax.md#pruning-blocks-retained)
+To use the ad-hoc tracing APIs, your node must be within the
```

Comment:
Does this make sense, or do we mean 'the requested block must be within the number of blocks retained with pruning enabled'?

Reply:
@rolandtyler, this still needs to be done (along with the other comments). Resolved this so that we can merge.